cis_proc.py

```diff
@@ -144,18 +146,18 @@ def main(tar_file, bids_dir, config, sub, ses=None, work_dir=None, n_procs=1):
     mriqc_work_dir = op.join(scan_work_dir, 'work')
 
     # Copy tar file to work_dir
-    if ses is None:
-        work_tar_file = op.join(scan_work_dir, 'sub-{0}.tar'.format(sub))
-    else:
-        work_tar_file = op.join(scan_work_dir, 'sub-{0}-ses-{1}.tar'.format(sub, ses))
+    work_tar_file = op.join(scan_work_dir, 'sub-{0}.tar'.format(sub))
```

Comment:
I reworked these to only modify the path if `ses` resolves to reduce some redundancy